content/09.evolution.md

```diff
@@ -341,10 +380,21 @@ Indeed, the E484K substitution seen in the P.1 and B.1.315 variants of the B.1.1
 For now, the consensus appears to be that the FDA-approved vaccines still seem to be generally effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948], though select VOCs may present challenges. 
 Further research is required to discern the clinical, prophylactic, and therapeutic consequences of these genetic SARS-CoV-2 variants as the pandemic evolves. 
 
-### Conclusions
+### Question 6: What is Next?
+
+The SARS-CoV-2 pandemic has presented many unprecedented scientific opportunities.
+The rapid identification of the genomic sequence of the virus allowed for early contextualization of SARS-CoV-2 among other known respiratory viruses, and the scientific community has continued to collect, analyze, and disseminate information about the SARS-CoV-2 virus and the associated illness, COVID-19 at previously unimaginable rates [@individual-methods].
+The accessibility of genome sequencing technology has allowed for deep sequencing of the virus to establish a level of viral surveillance that had never before been achieved [@doi:10/d5mg; @doi:10.1038/s41592-022-01444-z; @doi:10.1016/j.lanepe.2021.100210].
+The information obtained from genetic, bioinformatics, and evolutionary analysis has played a significant role in shaping the global pandemic response [@doi:10.1038/s41592-022-01444-z; @doi:10.1038/s41586-021-04188-6; @doi:10/hskp].
+<!-- Wastewater surveillance? Implications for preventing future pandemics? Vaccine response time?-->
```

Comment:
Maybe a good spot for the text @RLordan and @Sergey-Knyazev were thinking of adding?

Reply:
Yes, apologies, bare with me will get to it tomorrow

Reply:
No pressure @RLordan! You said you were busy through Tuesday but I just thought it might fit there so I thought I'd make a note-- and of course if you'd rather put it somewhere else, that is totally fine as well!

Reply:
@rando2 I have a few lines on this when this is merged :)